src/main/java/com/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,14 @@
+package com.bookstore.controller;
+
+import com.bookstore.dto.BookDto;
+import com.bookstore.dto.CreateBookRequestDto;
+import java.util.List;
+
+public interface BookController {
+
```

Comment:
I see that sometimes you put a new line here and sometimes you don't - please make sure that you keep your code consistent.